doc/src/modules/rewriting.rst

```diff
@@ -99,4 +99,4 @@ will be greatly improved.
 
 More information:
 
-.. autofunction:noindex: cse
+.. autofunction:: cse
```

Comment:
I think the noindex is supposed to prevent other references to `cse` from linking to here (instead of the main reference in `simplify.rst`).